docker/build.sh

```diff
@@ -2,7 +2,20 @@
 
 set -e
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+ARCHITECTURE="$(uname -m)"
 DOCKER_IMAGE_NAME="trellis-docker"
 
+case $ARCHITECTURE in
+    x86_64)
+        DOCKER_IMAGE_NAME+="amd64";;
```

Comment:
Looks like the colon is missing here. Needs to be `:amd64`. The intent is to make a string like `trellis-docker:amd64` which is `name:tag`

Reply:
resolved!